testresultparser/junit_reports.go

```diff
@@ -3,115 +3,134 @@ package testresultparser
 import "encoding/xml"
 
 type report struct {
-    TestSuite   string     `xml:"name,attr"`
-    Tests       int        `xml:"tests,attr"`
-    Failures    int        `xml:"failures,attr"`
-    Errors      int        `xml:"errors,attr"`
-    Skipped     int        `xml:"skipped,attr"`
-    Time        string     `xml:"time,attr"`
-    TestResults []testCase `xml:"testcase"`
+	TestSuite   string     `xml:"name,attr"`
+	Tests       int        `xml:"tests,attr"`
+	Failures    int        `xml:"failures,attr"`
+	Errors      int        `xml:"errors,attr"`
+	Skipped     int        `xml:"skipped,attr"`
+	Time        string     `xml:"time,attr"`
+	TestResults []testCase `xml:"testcase"`
+	StdOut      systemOut  `xml:"system-out"`
```

Comment:
In addition it would be great to have tests proving that the solution works. This should cover both Gradle where all system logs are in the same element as well as the case of surefire report where those outputs are next to the each test, but we still want to collect them in one field of our `report` structure.